defaults/main.yml

```diff
@@ -1,31 +1,30 @@
 ---
-php_version: "7.0"
-pid: /run/php-fpm.pid
-error_log: log/php-fpm.log
-log_level: notice
-syslog_facility: daemon
-syslog_ident: php-fpm
-emergency_restart_threshold: 0
-emergency_restart_interval: 0
-process_control_timeout: 0
-process_max: 0
-daemonize: Yes
-rlimit_files: 16400
-rlimit_core: 0
-systemd_interval: 10
-listen_backlog: -1
-listen_allowed_clients: any
-listen_mode: 0660
-pm_process_idle_timeout: 10s
-pm_max_requests: 0
-ping_response: pong
-request_terminate_timeout: 0
-request_slowlog_timeout: 0
-slowlog: log/php-fpm.log.slow
-catch_workers_output: no
-clear_env: Yes
-security_limit_extensions: .php .phar
-access_format: "%R - %u %t \"%m %r\" %s"
-php_ini_values:
-  expose_php:
-    - 0
+php_fpm_php_version: "7.0"
+php_fpm_pid: /run/php-fpm.pid
+php_fpm_error_log: log/php-fpm.log
+php_fpm_log_level: notice
+php_fpm_syslog_facility: daemon
+php_fpm_syslog_ident: php-fpm
+php_fpm_emergency_restart_threshold: 0
+php_fpm_emergency_restart_interval: 0
+php_fpm_process_control_timeout: 0
+php_fpm_process_max: 0
+php_fpm_daemonize: Yes
+php_fpm_rlimit_files: 16400
+php_fpm_rlimit_core: 0
+php_fpm_systemd_interval: 10
+php_fpm_listen_backlog: -1
+php_fpm_listen_allowed_clients: any
+php_fpm_listen_mode: 0660
+php_fpm_pm_process_idle_timeout: 10s
+php_fpm_pm_max_requests: 0
+php_fpm_ping_response: pong
+php_fpm_request_terminate_timeout: 0
+php_fpm_request_slowlog_timeout: 0
+php_fpm_slowlog: log/php-fpm.log.slow
+php_fpm_catch_workers_output: no
+php_fpm_clear_env: Yes
+php_fpm_security_limit_extensions: .php .phar
+php_fpm_access_format: "%R - %u %t \"%m %r\" %s"
+php_fpm_php_ini_values:
+  expose_php: 0
```

Comment:
This used to be a list, is this intended?

Reply:
it was where do you take that from?

Reply:
'Changed Files' shows the difference.